.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+on: [ push ]
+
+jobs:
+  migration_job:
+    runs-on: ubuntu-latest
+    name: CI migration
+    steps:
+      # Checkout the source code
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Run the migration with Postgres.ai Action
+      - name: DB migration with DLE
+        uses: agneum/migration-action@v0.0.10
+        id: db-migration
+        with:
+          dbname: test
```

Comment:
@agneum is this the dbname of the clone that will be generated on test run?

Reply:
Right